### ISSUE TITLE
Public head of master is failing windows CMAKE tests (?)

### ISSUE THREAD

**Issue by DavidNorman** [tensorflow/tensorflow#10673]
The following tests are failing for several of the public pull requests.  I suggest that the current head of master would also fail in the same way.

```
The following tests FAILED:
	171 - C:/tf_jenkins/home/workspace/tensorflow-pr-win-cmake-py/tensorflow/python/kernel_tests/sparse_ops_test.py (Failed)
	173 - C:/tf_jenkins/home/workspace/tensorflow-pr-win-cmake-py/tensorflow/python/kernel_tests/sparse_reshape_op_test.py (Failed)
```

**Comment by DavidNorman** [comment]
I'll revise that.  Looking at the history of the Windows CMAKE tests, I can see that this test has been failing on and off for quite a while (didn't go back to the origin to find out who actually broke it)

**Comment by aselle** [comment]
@yifeif, could you take a look?

**Comment by yifeif** [comment]
Flake was due to a new numpy version. Should be fixed with #10709 and @gunan re-downloaded numpy afterward.
